README.md

```diff
@@ -86,11 +86,12 @@ s = A[row_index, col_index].value           # extract single element
 ```
 ## Assign
 ```python
-M[rows, cols](mask, accum) << A             # rows and cols are a list or a slice
-M[rows, col_index](mask, accum) << v        # assign column
-M[row_index, cols](mask, accum) << v        # assign row
-M[rows, cols](mask, accum) << s             # assign scalar to many elements
-M[row_index, col_index] << s                # assign scalar to single element (mask and accum not allowed)
```

Comment:
I moved the commented text here to two lines, because it was too long and created a scroll box when viewed on github.